pdata/internal/common.go

```diff
@@ -500,7 +487,7 @@ func (v Value) asRaw() interface{} {
 	case ValueTypeInt:
 		return v.IntVal()
 	case ValueTypeBytes:
-		return v.BytesVal()
+		return v.BytesVal().Value()
```

Comment:
For the future, it appears one of the common places we call asRaw() from is when we need to convert it to to JSON string in `AsString()`. We can have a non-copying version of asRaw() to be used in `AsString()`. Can be a future PR (we can postpone it until we really see the need because of bad performance).

Reply:
Sounds good. Another option can be to introduce `Value.AsJSON() method